src/component/Card/ProductCard.tsx

```diff
@@ -0,0 +1,126 @@
+import React, {  useState } from 'react'
+import { Button, CardOuterDiv, H3Price } from './ProductCardStyled'
+import style from './ProductCard.module.css'
+import { useDispatch } from 'react-redux'
+import { notify } from '../Utils/Notify'
+import axios, { AxiosResponse } from 'axios'
+import { useNavigate } from 'react-router-dom'
+
+interface Props {
+  deleteHandler: { fn: (i: number) => void; identifier: number ,resource: string;}
+  title: string
+  image: string
+  description: string
+  id: number
+  price: number
+  wishListed?: boolean
+  addToWishListed?: { fn: (i: number) => void; identifier: number }
+  addToCart?: () => {}
+}
+export default function ProductCard(props: Props) {
+  const {
+    deleteHandler,
+    title,
+    image,
+    description,
+    id,
+    price,
+    wishListed,
+    
+    addToWishListed,
+    
+  } = props
+  
+  const [open, setOpen] = useState(false)
+  const dispatch = useDispatch()
+  const handleWhishList = () => {
+    // axios.put(process.env.REACT_APP_BASE_URL + 'products/' + id)
+    addToWishListed?.fn(addToWishListed.identifier)
+   
+  }
+  const getPromise = (
+    url: string,
+    promiseType: string
+  ): Promise<AxiosResponse<any>> => {
+    if (promiseType === 'post') {
+      return axios.post(url, {
+        id: id,
+        title: title,
+        price: price,
+        description: description,
+        image: image,
+      })
+    } else if (promiseType === 'delete') {
+      return axios.delete(url)
+    } else return axios.get(url)
+  }
+  const addToCartHandler = () => {
+
+    notify(
+      getPromise(process.env.REACT_APP_BASE_URL + 'cart/', 'post'),
+      ' Trying to add item to cart',
+      '🛒 Added to cart!👌'
```

Comment:
@rohantor constant strings shall be maintained in separate files.
Example:
"Trying to add item to cart"
"🛒 Added to cart!👌"

shall be moved to other file which is solely responsible for maintaining it.